src/CalcManager/pch.h

```diff
@@ -24,6 +22,29 @@
 #include <list>
 #include <regex>
 #include <unordered_map>
-#include <intsafe.h>
 #include <array>
+#include <string_view>
+
+#if defined(_WIN32) && defined(_MSC_VER)
+
+#include <windows.h>
+#include <winerror.h>
+#include <intsafe.h>
```

Comment:
```suggestion
```
This should not be required anymore, if you provide the `HIDWORD` and `LODWORD` macros for all the platforms

This should also let you add Ratpack library. See https://github.com/Microsoft/calculator/pull/398 where I remove this include

Reply:
@janisozaur Have so seen https://github.com/Microsoft/calculator/pull/211#discussion_r269383211 and https://github.com/fwcd/WindowsCalculator/pull/2? The issue with Ratpack ist actually a bit deeper than that.

Reply:
`-fms-extension` does not provide support for C++ flexible array members